buildconfig/Jenkins/conda-setup

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# This script expects a environment variable called $WORKSPACE to be set.
+
+# From https://stackoverflow.com/a/9107028 by https://stackoverflow.com/users/1184238/andrew-norrie
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+EXPECTED_MAMBAFORGE_PATH=$HOME/mambaforge
+EXPECTED_CONDA_PATH=$EXPECTED_MAMBAFORGE_PATH/bin/conda
+ENV_NAME="mantid-developer"
+
+MAMBAFORGE_SCRIPT_NAME=Mambaforge-$(uname)-$(uname -m).sh
+
+# Ensure conda is installed
+if [[ ! -f $EXPECTED_CONDA_PATH ]]; then
+    if [[ ! -f $MAMBAFORGE_SCRIPT_NAME ]]; then
+        # Download mambaforge
+        URL=https://github.com/conda-forge/miniforge/releases/latest/download/$MAMBAFORGE_SCRIPT_NAME
```

Comment:
I would move this to the top with the other constant variables.

Reply:
Moved